RELEASE.md

```diff
@@ -0,0 +1,256 @@
+            n. run `python3 ../../utils/cluster_manager.py -r 1 -n 3 --cluster-mode`
+                and
+                `python3 ../../utils/cluster_manager.py -r 1 -n 1`
+
+            o. Edit the file [standalone_example.py](https://github.com/valkey-io/valkey-glide/blob/main/examples/python/standalone_example.py) or [cluster_example.py](https://github.com/valkey-io/valkey-glide/blob/main/examples/python/standalone_example.py) to have the right port in the different functions and run python3.
+            p. Once the CD workflow and all test have passed, you can release a real version such as `git tag v1.3.0`.
+
+            q. Find how you saved the upstream remote repo (valkey-io/valkey-glide):
+
+            r.
+            ```
+            [10:09:23] ubuntu (v1.x*)$ git remote -vv
+            origin  https://github.com/your-repo/valkey-glide (fetch)
+            origin  https://github.com/your-repo/valkey-glide (push)
+            origin-aws      https://github.com/valkey-io/valkey-glide (fetch)
+            origin-aws      https://github.com/valkey-io/valkey-glide (push)
+            ```
+            s. Push the tag to the upstream remote so it will trigger the CD action:
+
+            t. git push origin-aws v1.x.x
+
+
+2. Update the release notes in github
+
+    a. Go to: https://github.com/valkey-io/valkey-glide/releases
+
+    b. Click on "Draft a new release"
+
+    c. Add the release tag
+
+    d. Copy + paste the release notes from the CHANGELOG.md file
+
+## How to use GLIDE's CD workflows
+
+For deployment guide, see the above section **`GLIDE - How to release a new version`**.
+
+### **Secrets**
+
+Please **contact a Valkey-GLIDE maintainer** to access the secrets for this repository.
+
+### **How to trigger the CD workflow**
+
+The CD workflow can be triggered in several ways.
+
+1. **By pushing a Version Tag**: Trigger the CD workflow by pushing a version tag in the format "v*.*." or "v.*.*-rc".
+For example, v1.0.0-rc1. When triggered by a push event with a version tag, all package manager CD workflows (e.g., pypi-cd.yml, npm-cd.yml, java-cd.yml) will be initiated.
+The version will be extracted from the tag.
+
+    a.
+    ```
+    [13:39:23] ubuntu $ git remote -vv
+    origin  https://github.com/your-repo/valkey-glide (fetch)
+    origin  https://github.com/your-repo/valkey-glide (push)
+    origin-aws      https://github.com/valkey-io/valkey-glide (fetch)
+    origin-aws      https://github.com/valkey-io/valkey-glide (push)
+    ```
+
+
+2. **By Workflow Dispatch:** Manually trigger the CD workflow using the "workflow dispatch" option.
+This is done on a specific CD workflow by passing the required version as an input parameter in the format "*.*.*" or "*.*.*-rc*".
+This method will not trigger deployments for other package managers and will not create a tag in the repository.
+
+    a. <br>
+    ![pypi CD](images-release-md/pypi-CD.png)
+
+
```

Comment:
TLDR below

You need to decide who is your target audience. Half of the instructions could be replaced by "ask maintainer to do ...". On another hand, anyone could help with testing RC and updating release notes.

I think you need to split this file into sections: one for an arbitrary open source contributor who wants to help with the release activities and other section for maintainers.
Maintainers don't need detailed instructions on how to run a CD job or create/push a branch.